Source/CarthageKit/VersionFile.swift

```diff
@@ -303,21 +304,35 @@ public func createVersionFileForCommitish(
 		platformCaches[platform.rawValue] = []
 	}
 
+	struct FrameworkDetail {
```

Comment:
private?

Reply:
This struct is defined in a method. So it is not possible to add access control.

Reply:
Didn't see that. Thanks!